main.go

```diff
@@ -554,6 +554,72 @@ var testCases = []testRun{
 			return metrics
 		},
 	},
+	{
+		name: "histogram with 1000 bucket bounds",
```

Comment:
we drop if bucket bounds are greater than 32 and there is a test case for this already. Is there some specific thing you are trying to check with this?

Reply:
Yes, if this ever changes, but we still store the bucket counts somewhere, I want to make sure we explicitly drop that.